assets/js/components/product-preview/index.js

```diff
@@ -47,7 +47,6 @@ const ProductPreview = ( { product } ) => {
 	return (
 		<div
 			className="wc-product-preview wc-block-grid__product"
-			style={ { maxWidth: `${ wc_product_block_data.thumbnail_size }px` } }
```

Comment:
You'll need to regenerate the snapshot for this component since you made this change (that's why the test is failing)– to run the tests locally, you can do `npm run test`, and to update the snapshot, `npm run test:update`